### ISSUE TITLE
[Bug]: Conversion to m4b canceled

### ISSUE THREAD

**Issue by mrburns-42** [advplyr/audiobookshelf#2390]
### Describe the issue

Since a few days the process to convert multiple mp3/m4b files to a m4b file sometimes aborts. Two examples:

1. yesterday I wanted to convert four audio books at the same time, with three of them everything worked without problems, with one the process was aborted.

2. today I wanted to convert two other audiobooks, none of them finished converting, the processes were terminated automatically at some point.

Restarting the Docker container did not help.

### Steps to reproduce the issue

1. Start the conversion of mp3/m4b files to a single m4b file.
2. After some time, the conversion process seems to be finished.
3. The books are sometimes converted, sometimes not.

### Audiobookshelf version

2.6.0

### How are you running audiobookshelf?

Docker

**Comment by advplyr** [comment]
Can you share the docker container logs?

**Comment by skronenwerth** [comment]
Not really. Is it possible to adjust the log level? Then I would deliberately convert some audiobooks in the next few days and wait until the error reappears. Then I could create another log file and make it available.

**Comment by advplyr** [comment]
If you go to the logs page in settings there is a dropdown you can select "Debug"

**Comment by skronenwerth** [comment]
Unfortunately, I haven't had time to test it extensively yet, but this will change shortly. I'll be in touch.

**Comment by skronenwerth** [comment]
[logs.txt](https://github.com/advplyr/audiobookshelf/files/13767386/logs.txt)

Here is a new log file. On December 24th I tried to build the audiobook "Old Man", which was aborted with an error message. The initial situation was two m4b files that were to be converted into a single file. The second attempt worked.

Today I wanted to convert the audiobook Lillian, which consisted of numerous mp3 files. In the end, the m4 file was slightly shorter than 60 minutes, but the mp3 files had a total length of more than 15 hours.

I am of the opinion that the error, which is why I opened the bug report, was different from the two examples now described. In the example with the audiobook "Old Man", I received an error message in the user interface, which was not the case with the other attempts. Also, no m4b file came out at all at the end.

I will continue to monitor this.

**Comment by mrburns-42** [comment]
I have now converted around 50 audiobooks since my last post. My described error has not occurred again ... so I can't reproduce it myself.

The error that occurred was that sometimes long audiobooks could not be converted correctly. When the m4b file is finished, it only has one or two minutes of listening time instead of about 40 hours. I believe that I have already found this error in another open issue ... this error occurs regularly.
